packages/core/build/config.js

```diff
@@ -22,6 +22,7 @@ const copyConfig = (base) => ([
     { from: path.resolve(__dirname, '../src/public/images/favicons/**') },
     { from: path.resolve(__dirname, '../src/public/images/common/logos/platform_logos/**') },
     { from: path.resolve(__dirname, '../src/public/images/app/header/**') },
+    { from: path.resolve(__dirname, '../node_modules/deriv-components/lib/icon/sprite'), to: 'sprite', toType: 'dir' },
```

Comment:
Can we structure this inside the `public` folder that is created?

Reply:
sure, we can